projects/action-bars-showcase/src/app/app-routing.module.ts

```diff
@@ -18,10 +17,6 @@ const routes: Routes = [
     path: 'visual/tab-summary-action-bar',
     component: SkyTabSummaryActionBarDemoComponent,
   },
-  {
-    path: 'visual/split-view-summary-action-bar',
-    component: SplitViewSummaryActionBarVisualComponent,
-  },
```

Comment:
☝️ Unrelated to this PR.... a circular dependency was removed from split-view in an earlier PR, but these files were accidentally left in the code base. Wouldn't build without removing them.